test/e2e/test_auditlogs.go

```diff
@@ -31,11 +31,6 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
-const (
```

Comment:
These two constants are used both in CloudAuditLogsSourceWithTestImpl of test/e2e/test_auditlogs.go and BrokerEventTransformationTestWithAuditLogsSourceHelper of test/e2e/lib/helpers/broker_event_transformation_test_with_source_helper.go.  I move these two constants to libs because clients use the same args.